### DISCUSSION TITLE
Support for the m5stack's 4-relay

### DISCUSSION THREAD

**Discussion by Dennis3321** [raomin/ESPAltherma#291] [Ideas]
My Altherma can heat and cool. So in total I need 4 relays to control everything;
relay 1; heating on / off
relay 2; cooling on / off
relay 3; smart grid contact 1
relay 4; smart grid contact 2

So I've decided to re-write the software to be complatible with the with the m5stack's 4-relay hardware. A bit more expensive than the aliexpress relay's, but it's still around 15 euro's. But far better standardized. 

I've already at 50 percent with the coding, so if someone is interested...

**Comment by nolantern** [comment]
I am interested to use this for the power limit feature.
Can you share work?
